test/rendering/snapshots.spec.js

```diff
@@ -0,0 +1,15 @@
+import { test, expect } from '@playwright/test';
+
+test.describe('should render', () => {
+  const content = '.sn-org-chart';
+
+  test.describe('from snapshot', () => {
+    test('basic example', async ({ page }) => {
+      await page.goto('/render?snapshot=basic', { waitUntil: 'networkidle' });
+      await page.waitForSelector('.sn-org-chart-snapshot', { visible: true });
+      const elem = await page.$(content);
+      const img = await elem.screenshot();
+      expect(img).toMatchSnapshot('basic-snapshot.png');
```

Comment:
I learned that toMatchSnapshot() now should be used for comparing text or arbitrary binary data
to produce and visually compare screenshots using await expect(page).toHaveScreenshot().
https://playwright.dev/docs/test-snapshots
I did that in https://github.com/qlik-oss/sn-table/pull/988/files#diff-d8d306b0b30eef9dd807a03675987a618b0b89a804d01988bd7dbc8afb215c8fR72

Reply:
Oh that is good to know. I'll update!